argo-cd-apps/base/monitoring-workload-in-cluster.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Create also record in argo-cd-apps/overlays/development/repo-overlay.yaml:
```
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: monitoring-workload-in-cluster
spec:
  source:
    repoURL: https://github.com/redhat-appstudio/infra-deployments.git
    targetRevision: main
```

Reply:
done